src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -51,16 +51,15 @@ public enum QuicError
         InternalError = 1,
         ConnectionAborted = 2,
         StreamAborted = 3,
-        AddressInUse = 4,
-        InvalidAddress = 5,
         ConnectionTimeout = 6,
-        HostUnreachable = 7,
         ConnectionRefused = 8,
         VersionNegotiationError = 9,
         ConnectionIdle = 10,
         ProtocolError = 11,
         OperationAborted = 12,
         AlpnInUse = 13,
+        TransportError = 14,
+        UserCallbackError = 15,
```

Comment:
It was approved just as `CallbackError`: https://github.com/dotnet/runtime/issues/87259#issuecomment-1603037411

Reply:
good catch. updated.